src/org/thoughtcrime/securesms/notifications/NotificationCenter.java

```diff
@@ -0,0 +1,485 @@
+                builder.setContentTitle(dcChat.getName());
+            }
+
+            // if privacy allows, for better accessibility,
+            // prepend the sender in the ticker also for one-to-one chats (for group-chats, this is already done)
+            String tickerLine = line;
+            if (!dcChat.isGroup() && privacy.isDisplayContact()) {
+                line = dcContext.getContact(dcMsg.getFromId()).getFirstName() + ": " + line;
+            }
+            builder.setTicker(tickerLine);
+
+            // set sound, vibrate, led for systems that do not have notification channels
+            if (!notificationChannelsSupported()) {
+                if (signal) {
+                    Uri sound = effectiveSound(chatId);
+                    if (sound != null) {
+                        builder.setSound(sound);
+                    }
+                    boolean vibrate = effectiveVibrate(chatId);
+                    if (vibrate) {
+                        builder.setDefaults(Notification.DEFAULT_VIBRATE);
+                    }
+                }
+                String ledColor = Prefs.getNotificationLedColor(context);
+                if (!ledColor.equals("none")) {
+                    builder.setLights(getLedArgb(ledColor),500, 2000);
+                }
+            }
+
+            // set avatar
+            Recipient recipient = new Recipient(context, dcChat, null);
+            if (privacy.isDisplayContact()) {
+                try {
+                    Drawable drawable;
+                    ContactPhoto contactPhoto = recipient.getContactPhoto(context);
+                    if (contactPhoto != null) {
+                        drawable = GlideApp.with(context.getApplicationContext())
+                                .load(contactPhoto)
+                                .diskCacheStrategy(DiskCacheStrategy.NONE)
+                                .circleCrop()
+                                .submit(context.getResources().getDimensionPixelSize(android.R.dimen.notification_large_icon_width),
+                                        context.getResources().getDimensionPixelSize(android.R.dimen.notification_large_icon_height))
+                                .get();
+
+                    } else {
+                        drawable = recipient.getFallbackContactPhoto().asDrawable(context, recipient.getFallbackAvatarColor(context));
+                    }
+                    if (drawable != null) {
+                        int wh = context.getResources().getDimensionPixelSize(R.dimen.contact_photo_target_size);
+                        Bitmap bitmap = BitmapUtil.createFromDrawable(drawable, wh, wh);
+                        if (bitmap != null) {
+                            builder.setLargeIcon(bitmap);
+                        }
+                    }
+                } catch (Exception e) { Log.w(TAG, e); }
+            }
+
+            // add a reply-button that allows answering without opening Delta Chat.
```

Comment:
The "Mark read" button in the notifcations is gone :disappointed:

Reply:
yip, this is by design, it was never really correct the old way.

"markread" means we send out an mdn, this should really only happy when the whole message is at least displayed once to the user,  this is not true for an excerpt shown in the notification area. 

you can swipe the notification away if you want to dismiss it locally.

Reply:
Maybe we could at least have a markread button when the whole message is shown in the notification?

Reply:
hm, then i'd do it unconditionally. do you know if whatsapp/telegram mark things already as read one the sender side if the user hits such a button in the notification?

having the button sometimes only would introduce even more questions.

but if this is the last open question of this pr, i'd like to discuss/change this in a follow up, so that these changes can already be tested in the nightlies.

Reply:
but wait: the old "Mark read" button did not really sent out read-receipts, it only removed the unread-counter, i'd say this is a things we can do, you're right.

Reply:
re-added mark-read in https://github.com/deltachat/deltachat-android/pull/1368/commits/e8a3bae004f730ee5499cca657dd5f90b0faf079